README.md

```diff
@@ -1 +1,83 @@
-# git
\ No newline at end of file
+<!-- Headings -->
+# Sample .md file
+## This is a sub heading
+### Another sub heading 
+---
+
+<!-- italics-->
+_This text in italic_
+___
+
+<!--bold-->
+__This text in bold__
+___
+
+<!--strikethrough-->
+~~Strikethrough~~
+___
+
+<!--Horizontal Rule-->
+<!--- ___>
+
+<!--BlockQuote-->
+>This is a block quote
+___
+
+<!--Links-->
+[GitHub](https://github.com/KaranVyas "GitHub profile for Karan Vyas")
```

Comment:
This doesn't look good, we need something more clean.

Reply:
I have gone through many blogs, but unable to figure a clean version for this

Reply:
Done.